tools/src/main/scala/scala/scalanative/linker/Infos.scala

```diff
@@ -7,6 +7,7 @@ import scalanative.nir._
 sealed abstract class Info {
   def attrs: Attrs
   def name: Global
+  implicit def position: Position
```

Comment:
That this is implicit is very confusing. Consider:
```suggestion
  def position: Position
```